src/general/MusicContext.cs

```diff
@@ -6,5 +6,7 @@ public enum MusicContext
 {
     PatchVents = 1,
 
+    PatchTidepool = 2,
```

Comment:
For consistency this shouldn't specify the value as PatchIceShelf doesn't either.